sig-data-science/README.md

```diff
@@ -34,8 +34,14 @@ The Chairs of the SIG run operations and processes governing the SIG.
 
 The following [subprojects][subproject-definition] are owned by sig-data-science:
 ### AIOps
-- **Owners:**
+**Owners:**
   - [open-services-group/community/sig-data-science](https://github.com/open-services-group/community/blob/main/sig-data-science/OWNERS)
+
+  #### 1. Operate First Data Science Community
+   - The Operate First Data Science Community aims to embrace the multidisciplinary nature of the current state of AI operations and cloud infrastructure by discussing technical topics relevant to data scientists, software developers, DevOps professionals, statisticians, and the intersection of their work, all in an open and collaborative public platform.
+
```

Comment:
This document is auto-generated so placing content here might mess it up.  You will need to add a section below and move these subproject details there. please see this example:  https://github.com/kubernetes/community/blob/master/sig-architecture/README.md

Reply:
You add content to https://github.com/open-services-group/community/blob/main/sigs.yaml#L30
and then this template is used
https://github.com/open-services-group/community/blob/main/generator/sig_readme.tmpl

see the template for available variables

Reply:
I dont think the the subproject description content is auto generated by sig_readme.tmpl. See the `<!-- BEGIN CUSTOM CONTENT -->` line below the generated content here, https://raw.githubusercontent.com/kubernetes/community/master/sig-architecture/README.md

Reply:
Here is some more docs on how to add custom content to the README: 
https://github.com/kubernetes/community/blob/master/generator/README.md#adding-custom-content 

@aakankshaduggal can you please move this content to the "custom content" section of the readme?

Reply:
Done :+1: